RELEASE/scripts/autoscend/quests/level_9.ash

```diff
@@ -315,7 +315,7 @@ boolean L9_aBooPeak()
 	{
 		auto_log_info("A-Boo Peak (initial): " + get_property("booPeakProgress"), "blue");
 
-		if (clueAmt < 3 && item_amount($item[January\'s Garbage Tote]) > 0)
+		if (clueAmt < 3 && item_amount($item[January\'s Garbage Tote]) > 0 && auto_is_valid($item[Broken Champagne Bottle]))
 		{
 			januaryToteAcquire($item[Broken Champagne Bottle]);
 			removeFromMaximize("-equip " + $item[Broken Champagne Bottle]);
```

Comment:
```suggestion
		if (clueAmt < 3)
		{
			januaryToteAcquire($item[Broken Champagne Bottle]);
```

`januaryToteAcquire()` calls `isjanuaryToteAvailable()` which checks `auto_is_valid($item[January\'s Garbage Tote])` so it should cover all paths and it handles the `removeFromMaximize()` call too.

Reply:
much better solution, committed